apps/storefront-e2e/src/integration/ssr.cy.ts

```diff
@@ -30,26 +65,67 @@ describe('SSR suite', { tags: 'smoke' }, () => {
     pdp.getImages().should('be.visible');
     pdp.getDescription().should('be.visible');
     pdp.getAttributeTerms().should('have.length', 7);
+    verifyFooter();
   });
 
   it('must render Contact us page', () => {
     const contactPage = new ContactPage();
 
     contactPage.visit();
+
+    verifyHeader();
+
     contactPage.getHeading().should('be.visible');
+    verifyFooter(false);
   });
 
   it('must render Login page', () => {
     const loginPage = new LoginPage();
 
     loginPage.visit();
+
+    verifyHeader();
+
     loginPage.loginForm.getWrapper().should('be.visible');
+    verifyFooter();
   });
 
   it('must render Cart page', () => {
     const cartPage = new CartPage();
 
     cartPage.visit();
+
+    verifyHeader();
+
     cartPage.getCartEntriesWrapper().should('be.visible');
+    verifyFooter();
+  });
+
+  it('must render Search page', () => {
+    const searchPage = new SearchPage();
+
+    searchPage.visit();
+
+    verifyHeader();
+
+    searchPage.getFacets().should('be.visible');
+    searchPage.getProductSort().should('be.visible');
+    searchPage.getOryxCards().should('have.length.greaterThan', 1);
+    verifyFooter();
+  });
+
+  it('must render Category page', () => {
+    const categoryId = '6';
+    const categoryData = { id: categoryId };
```

Comment:
We can drop `categoryId` here

```suggestion
    const categoryData = { id: '6' };
```

Reply:
Done ✅